src/common/scss/variables/zindex.scss

```diff
@@ -0,0 +1,5 @@
+:root {
+  --kd-z-overlay: 100000;
+  --kd-z-header: 10000;
+  --kd-z-local-nav: 9000;
+}
```

Comment:
is this specific for **gridStack** or we can use this across the other components too?

Reply:
Ah yes I forgot about this. I wanted to create a z-index table of contents basically to help us choose values and prevent layering issues.